src/main/java/org/broadinstitute/hellbender/engine/spark/GATKSparkTool.java

```diff
@@ -57,13 +57,13 @@
     @ArgumentCollection
     protected IntervalArgumentCollection intervalArgumentCollection = requiresIntervals() ? new RequiredIntervalArgumentCollection() : new OptionalIntervalArgumentCollection();
 
-    private JavaSparkContext sparkContext;
-    private ReadsSparkSource readsSource;
-    private SAMFileHeader readsHeader;
+    private transient JavaSparkContext sparkContext;
+    private transient ReadsSparkSource readsSource;
+    private transient SAMFileHeader readsHeader;
     private String readInput;
-    private ReferenceMultiSource referenceSource;
-    private SAMSequenceDictionary referenceDictionary;
-    private List<SimpleInterval> intervals;
+    private transient ReferenceMultiSource referenceSource;
+    private transient SAMSequenceDictionary referenceDictionary;
+    private transient List<SimpleInterval> intervals;
```

Comment:
I can see cases where workers might want to call some of the functions in `GATKSparkTool`, so I don't think we can break them like this. I'd recommend deleting the `sparkContext` field from `GATKSparkTool` entirely, and modifying `runPipeline()` to pass the context in explicitly to the `initialize*` methods. That field was just a convenience anyway.